examples/ComputeAndAggregateJobPostingProperties.py

```diff
@@ -94,13 +96,13 @@
         grouping_properties=grouping_properties,
         aggregate_properties=aggregate_properties,
         aggregate_functions={'posting_id_present': [numpy.sum]},
-        aggregations_path='s3://test-bucket/aggregated_properties',
+        storage=s3_storage,
         aggregation_name='title_state_counts'
     )
 
-    s3 = s3fs.S3FileSystem()
     logging.info('Logging all rows in aggregate file')
-    with s3.open(aggregate_path, 'rb') as f:
+    s3 = s3fs.S3FileSystem()
```

Comment:
Could we just use the s3 storage object here? I guess it would involve creating a BytesIO out of the return value:
`csv.reader(BytesIO(s3_storage.load(aggregate_path)))`

It would be nice to get rid of as much of the s3fs-specific code as possible.